### ISSUE TITLE
Angular 9 compatibility

### ISSUE THREAD

**Issue by real-artswan** [syncfusion/ej2-angular-ui-components#90]
Just upgraded my project to Angular 9.
Scheduler component is not visible, it has width and height 100%, and even if I set them to some hardcoded values it is still not rendered. In Angular 8 everything is fine.
Does it support Angular 9? Is it a bug? Should I change something in my code/html?

**Comment by real-artswan** [comment]
![зображення](https://user-images.githubusercontent.com/3417195/74689676-c2c05780-51e4-11ea-934b-89a92dd9b9b1.png)
above is a screenshot from angulr8 build, as you can see it has children
![зображення](https://user-images.githubusercontent.com/3417195/74689762-0c10a700-51e5-11ea-89c5-7e8d334575ed.png)
above is a screenshot from angular9 build, as you can see it doesn't have any children

**Comment by VengateshManiraj** [comment]
Dear Customer, 

Greetings from Syncfusion Support. 

No. It does not support Angular 9. Currently, we are working on Angular 9 support and it will be available in any of our upcoming releases. We would appreciate your patience until then. 

Please let us know if you have any further assistance.  

Regards, 
Vengatesh

**Comment by VengateshManiraj** [comment]
Dear Customer,

We are glad that we have provided the Angular 9 support for our components and it could be available in the version of v17.4.51. So please upgrade your package version to v17.4.51 for this support and for your reference we have prepared the below sample in Angular 9.

Sample: https://www.syncfusion.com/downloads/support/directtrac/general/ze/ej2-angular-scheduler1010869924 

Kindly check the above sample and get back to us for further assistance.

Regards,
Vengatesh

**Comment by ecappa** [comment]
Same issue here, when upgrading to 9 with splitter view...
Nothing appear. :( - js V 17.4.52.

**Comment by rajendranr-5483** [comment]
Hi Eric,
We created a sample with splitter and schedule in angular 9 to reproduce your reported issue. But, we couldn't replicate the issue. 
Could you please share details to reproduce the issue along with your package.json details?
Regards,
Rajendran R.

**Comment by real-artswan** [comment]
Hello, I appreciate your work guys, but I still have a problems with the schedule component.
I updated to
```
"@syncfusion/ej2-angular-calendars": "^17.4.51",
"@syncfusion/ej2-angular-schedule": "^17.4.55",
```
And now I can see schedule grid is rendered, but I have nothing except header in the TimelineWeek view, and also it doesn't show any events in any view and time to time throws errors to console
```
ERROR InternalError: "too much recursion"
    deprecate util.js:76
    deprecate util.js:83
    deprecate util.js:83
    deprecate util.js:83
    deprecate util.js:83
```
and
```
ERROR TypeError: "this.element.querySelector(...) is null"
    getWorkCellWidth ej2-schedule.es2015.js:19860
    getLeftFromDateTime ej2-schedule.es2015.js:19856
    changeCurrentTimePosition ej2-schedule.es2015.js:19822
    currentTimeIndicatorTimer ej2-schedule.es2015.js:17449
core.js:6185:19
```
So do I need to make any changes in my code to migrate to Angular 9?

**Comment by sellakumark** [comment]
@real-artswan  - Sorry for the delayed response.

We are glad to inform Angular 9 migration-related defects have been resolved at our end and resources in TimelineWeek view have been working properly. For your reference, we had prepared the sample which is available in the below link,

https://stackblitz.com/edit/angular-684tjm?file=app.component.ts

Kindly try the above sample and get back to us for further assistance.
